src/components/Gradebook/index.jsx

```diff
@@ -343,6 +343,8 @@ export default class Gradebook extends React.Component {
     return headings;
   }
 
+  toggleFormat = format => (this.props.toggleFormat(format));
```

Comment:
This may be redundant; you should be able to pass in `this.props.toggleFormat` in place of `this.toggleFormat`.

To be clear there isn't an actual impact to this; only motivation is making the code more concise.